include/psp2kern/ctrl.h

```diff
@@ -38,7 +38,7 @@ enum SceCtrlPadButtons {
 	SCE_CTRL_CIRCLE      = 0x002000,	//!< Circle button.
 	SCE_CTRL_CROSS       = 0x004000,	//!< Cross button.
 	SCE_CTRL_SQUARE      = 0x008000,	//!< Square button.
-	SCE_CTRL_HP          = 0x080000,	//!< Headphone pressent.
+	SCE_CTRL_HP          = 0x080000,	//!< Headphone plugged in.
```

Comment:
Is `SCE_CTRL_HP` an official name? If it's not I think it's more clear to put `SCE_CTRL_HEADPHONE`.